cms/models.py

```diff
@@ -29,6 +29,18 @@ class Meta:
         blank=True,
         help_text="A short description indicating how long it takes to complete (e.g. '4 weeks')",
     )
+    time_commitment = models.CharField(
```

Comment:
@gsidebo I want to know one thing, either these fields should not be the part of course only instead of repeating in program too ? talking about ~ duration, time_commitment, design_format

Reply:
That depends on what the designs look like. I'm pretty sure we want both the program and course detail pages to include `duration` and `time_commitment`, so the fields should exist on both models.

I actually think you can remove the `design_format` field and hardcode the "Online" value. We talked about it a little in planning. We don't know of any text that would be used there besides "Online". I think we should add the field only when we know of some other text that might go there